.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "thirdparty/onnx"]
 	path = thirdparty/onnx
 	url = https://github.com/onnx/onnx.git
+[submodule "thirdparty/fp16"]
```

Comment:
i thought this is merged :)

Reply:
No yet. It will disappear when I rebase.